forge.config.js

```diff
@@ -1,8 +1,9 @@
 /* tslint:disable */
 
-const path = require('path');
-const package = require('./package.json');
+const path = require('path')
+const packackeJson = require('./package.json')
```

Comment:
Interesting rename.

Reply:
`package` is a reserved word in strict mode and I should have never used it 😢